src/js/vic-v2/components/CropperController.jsx

```diff
@@ -402,12 +417,22 @@ export default class CropperControls extends React.Component {
     });
   }
 
+  // Cropper doesn't support changing width after componentDidMount
+  // use the windowWidth as a key to for the component to remount when the window width changes
   render() {
+
+    // narrow-layout class controls visiblity of elements based on window width
+    // if layout is forced narrow, remove this behavior
+    const narrowLayoutButtonsClassNames = classNames('cropper-control-row', { 'narrow-layout': !this.props.forceNarrowLayout });
+
+    // prevent the cropper from re-mounting when forcing narrow layout and width is in large screen range
+    const cropperKey = this.props.forceNarrowLayout && this.state.windowWidth >= LARGE_SCREEN ? LARGE_SCREEN : this.state.windowWidth;
```

Comment:
This a little ugly but I wanted to prevent the cropper from remounting if the width of its container wasn't changing.